### DISCUSSION TITLE
How to give feedback on AppXManifest Schema Documentation (Desktop7:Shortcut)

### DISCUSSION THREAD

**Discussion by TimMangan** [microsoft/WindowsAppSDK#4493] [General]
It used to be that the link to provide feedback on documentation pages of the AppXManifest schemas went right to a feedback page specific to the page you were on.  Sure, things need to change once in a while and I recall seeing that this would change.

But the feedback link at the bottom of the page https://learn.microsoft.com/en-us/uwp/schemas/appxpackage/uapmanifestschema/element-desktop7-shortcut (and I suspect other document pages) brings me to the general WindowsAppSDK page and I cannot even find the documentation page I want to provide feedback in this github repo.

What's a guy with a bug to do?

**Comment by kmgallahan** [comment]
https://learn.microsoft.com/en-us/contribute/content/provide-feedback

https://techcommunity.microsoft.com/t5/microsoft-learn-blog/announcing-a-new-way-to-give-feedback-on-microsoft-learn/ba-p/4027635
